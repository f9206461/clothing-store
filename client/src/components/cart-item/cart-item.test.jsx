import React from "react";
import { shallow } from 'enzyme';
import CartItem from "./cart-item.component";

describe('cart-item', () => {
    const mockItem = {
        imageUrl: 'https://www.testImage.com', 
        price: 90, 
        name: 'jackets', 
        quantity: 2
    };
    const wrapper = shallow(<CartItem item={mockItem} />);

    it('should render CartItem component', () => {
        expect(wrapper).toMatchSnapshot();
    });
});