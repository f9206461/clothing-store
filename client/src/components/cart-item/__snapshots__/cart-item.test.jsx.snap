// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cart-item should render CartItem component 1`] = `
<CartItemContainer>
  <ImgContainer
    alt="item"
    src="https://www.testImage.com"
  />
  <ItemDetailsContainer>
    <NameContainer>
      jackets
    </NameContainer>
    <span>
      2
       x $
      90
    </span>
  </ItemDetailsContainer>
</CartItemContainer>
`;
